library/translation.inc.php

```diff
@@ -263,6 +263,15 @@ function html_header_show() {
     //  We commented this out since we are now standardizing encoding
     //  in the globals.php file.
```

Comment:
Is there a reason why you are keeping the function in translations.inc.php? Shouldn't this maybe be in it's own file or global.php?